src/components/ReportActionItem/ReportActionItemImage.tsx

```diff
@@ -1,47 +1,38 @@
 import Str from 'expensify-common/lib/str';
-import PropTypes from 'prop-types';
 import React from 'react';
+import type {ReactElement} from 'react';
 import {View} from 'react-native';
-import _ from 'underscore';
 import AttachmentModal from '@components/AttachmentModal';
 import EReceiptThumbnail from '@components/EReceiptThumbnail';
 import Image from '@components/Image';
 import PressableWithoutFocus from '@components/Pressable/PressableWithoutFocus';
 import {ShowContextMenuContext} from '@components/ShowContextMenuContext';
 import ThumbnailImage from '@components/ThumbnailImage';
-import transactionPropTypes from '@components/transactionPropTypes';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as TransactionUtils from '@libs/TransactionUtils';
 import tryResolveUrlFromApiRoot from '@libs/tryResolveUrlFromApiRoot';
 import CONST from '@src/CONST';
+import type {Transaction} from '@src/types/onyx';
 
-const propTypes = {
+type ReportActionItemImageProps = {
     /** thumbnail URI for the image */
-    thumbnail: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    thumbnail?: string | number;
 
     /** URI for the image or local numeric reference for the image  */
-    image: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+    image: string | number;
 
     /** whether or not to enable the image preview modal */
-    enablePreviewModal: PropTypes.bool,
+    enablePreviewModal?: boolean;
 
     /* The transaction associated with this image, if any. Passed for handling eReceipts. */
-    transaction: transactionPropTypes,
+    transaction?: Transaction;
 
     /** whether thumbnail is refer the local file or not */
-    isLocalFile: PropTypes.bool,
+    isLocalFile: boolean;
```

Comment:
Should this be optional?

Reply:
@FitseTLT please check this. With this clarified, we can wind this up.

Reply:
Done